src/commands/publish.ts

```diff
@@ -142,18 +152,24 @@ const newMajorVersion = async (): Promise<void> => {
 };
 
 const update = async ({
+    cacheDuration,
     togglePublic,
     unminified,
     updatedCategory,
     updatedIntegration,
 }: {
+    cacheDuration: string;
```

Comment:
Should this be `cacheDuration: string | undefined;`? There seem to be some checks if `cacheDuration` is truthy in `update()`